using the ASP.NET Core HealthChecks library for the liveness/readiness sample

areller opened the pull request:
As discussed in #502

jkotalik commented on docs/recipes/probes.md:
````diff
@@ -66,20 +66,35 @@ tye run
+    .AddCheck("someReadinessCheck", new MyGenericCheck(_statusDictionary, "someReadinessCheck"), failureStatus: HealthStatus.Unhealthy, tags: new[] { "readiness" });
+```
+
+(Starting .NET 5, the HealthChecks library ships together with ASP.NET Core)
````
I believe this package has been shipping since 2.2. I'm not sure if we are shipping the HealthChecks library with ASP.NET Core though.

areller replied:
you're right. I saw that the HealthChecks library moved to the `aspnetcore` repository after .NET 5, but it has been and still part of the extensions library.

davidfowl commented:
Thanks for the doc! I think we should leave the readiness check and liveness check as fake checks that don't change as a result of an http request. It's fine in a sample but I would leave it out of the doc since it's not exactly what you would do in practice.

areller commented:
@davidfowl  I removed the whole talk about the `/set` endpoint away from the doc, but I added it to the `/` endpoint of the sample service. how's that?

jkotalik commented:
LGTM, thanks!
